history/cultures/0_DATES_README.txt

```diff
@@ -0,0 +1,13 @@
+1.1.1 (-500) - START, LEVIRTOTH IMPROSIONED
```

Comment:
This branch should be based on the culture history.
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request